rotkehlchen/chain/substrate/manager.py

```diff
@@ -26,6 +25,7 @@
 from rotkehlchen.user_messages import MessagesAggregator
 from rotkehlchen.utils.serialization import jsonloads_dict
 
+from ...db.settings import CachedSettings
```

Comment:
don't do relative imports like this unless it's meant to be part of the same module/directory.

```suggestion
from rotkehlchen.db.settings import CachedSettings
```

Reply:
Ahh, this was done by the Pycharm automatically when I moved the CachedSettings

Reply:
Please don't use any external configurations apart from our own configs for all linting programs we use. This is what make lint does. This is how we guarantee consistency across the codebase.